<launch>
  <arg name="fix_topic"               doc="Topic name of the NavSatFix message"/>
  <arg name="output_path"             doc="Output file path" />
  
  <node name="navsatfix_exporter" pkg="kimera_distributed" type="navsatfix_export_node" output="screen">
    <param name="~fix_topic"  value="$(arg fix_topic)" />
    <param name="~output_path"  value="$(arg output_path)" />
  </node>

</launch>